src/components/delete-track-dialog.tsx

```diff
@@ -0,0 +1,47 @@
+import {
+    Box,
+    Button,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogTitle,
+    FormControl,
+    FormHelperText,
+    TextField,
+} from "@material-ui/core";
+import { Track } from "../store/types";
+
+export interface DeleteTrackBoxProps {
+    isOpen: boolean;
+    onClose: () => void;
+    track: Track;
+    onSubmit: () => void;
+}
+
+export default function DeleteTrackDialog({
+    isOpen,
+    onClose,
+    track,
+    onSubmit,
+}: DeleteTrackBoxProps) {
+    const handleSubmit = () => {
+        console.log("yo");
+        onSubmit();
+    };
+
+    return (
+        <Dialog open={isOpen} onClose={onClose}>
+            <DialogTitle>Permanently delete {track.name}?</DialogTitle>
+            <Button variant="contained" onClick={onClose}>
+                Cancel
+            </Button>
+            <Button
+                variant="contained"
+                color="secondary"
+                onClick={handleSubmit}
+            >
+                yeet dat shit
+            </Button>
+        </Dialog>
```

Comment:
checkout the MaterialUI `Dialog` related components, like `DialogContent` and `DialogActions`